deployment/CE04OSBP_Deploy.csv

```diff
@@ -1,44 +1,44 @@
-CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
```

Comment:
Just this last file for line endings correction. Otherwise everything looks good.